algs/OdMatrixFromLayersAsLines.py

```diff
@@ -126,6 +128,10 @@ def initAlgorithm(self, config=None):
                            self.tr('Fastest Path (time optimization)')
                            ]
 
+        self.PATH_TYPES = [self.tr('Straight  Line (as the crow flies)'),
+                           self.tr('Line Follows Path')
```

Comment:
I think it would be better to adjust the description of the options a little. What do you think about `Matrix geometry follows straight lines (as the crow flies)` and `Matrix geometry follows routes`. In this case I would also rename the parameter to `Matrix geometry type`

Reply:
should be a simple change. Ill edit and send a new PR

Reply:
no need to open a new one - just add a new commit to this PR.

Reply:
sorry.. that's what I meant!